server/test/controllers/assessmentModel.test.ts

```diff
@@ -152,6 +156,28 @@ describe(
       expect(res.body.data.assessmentModel.id).toBeDefined();
     });
 
+    it(
+      'should add an assessment model when course exists (teacher in charge of the course)',
+      async () => {
+        (TeacherInCharge.findOne as jest.Mock).mockResolvedValueOnce({});
```

Comment:
Didn't we agree to use `jest.spyOn` for mocking return values?

Reply:
fixed